packages/web3-eth-accounts/src/index.js

```diff
@@ -42,14 +42,14 @@ var isNot = function(value) {
 };
 
 var trimLeadingZero = function (hex) {
-    while (hex && hex.startsWith('0x00')) {
-        hex = '0x' + hex.slice(4);
+    while (hex && hex.startsWith('0x0')) {
```

Comment:
The problem with this is what if the rs, value is `0x00001234` Then a even number is made uneven